lib/automation.rb

```diff
@@ -19,11 +19,14 @@ module Automation
         id: "mistralai/Mistral-7B-Instruct-v0.2",
         name: "discourse_automation.ai_models.mistral_7b_instruct_v0_2",
       },
+      { id: "command-r", name: "discourse_automation.ai_models.command_r" },
+      { id: "command-r-plus", name: "discourse_automation.ai_models.command_r_plus" },
     ]
 
     def self.translate_model(model)
       return "google:gemini-pro" if model == "gemini-pro"
       return "open_ai:#{model}" if model.start_with? "gpt"
+      return "cohere:#{model}" if model.start_with? "command"
```

Comment:
My only comment would be that, maybe we prefix these command models with `cohere_`/`cohere-` then this `model.start_with?` will be more specific; maybe some other model will start with command later on? Anyway not a big deal

Reply:
yeah this is likely going to change over the next few months... this whole pattern is not great